setupmeta/versioning.py

```diff
@@ -233,7 +233,7 @@ def rendered(self, version, extra=True):
             bits = list(bits)
             last = bits[-1]
             prelast = bits[-2]
-            if last and last.text == "dev" and prelast and prelast.text in BUMPABLE:
+            if last and (last.text == "dev" or last.text == "devcommit") and prelast and prelast.text in BUMPABLE:
```

Comment:
Note for later: come up with a better way to deal with "pre-defined" terms like that 😃 
Like `if last in DEV_STRATEGIES and ...`